package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "license": "GPL-3.0-or-later",
+  "private": true,
+  "browserslist": [
+    "> 0.5%",
+    "last 2 versions",
+    "Firefox ESR",
+    "not ie <= 10",
+    "not ie_mob <= 10",
+    "not bb <= 10",
+    "not op_mob <= 12.1"
+  ],
+  "scripts": {
+    "start": "webpack --watch",
```

Comment:
I would expect to call this `watch` but this may be my personal experiecne. I am fine if others call it differently, just give this name one more thought.